crates/miden-lib/src/account/interface/component.rs

```diff
@@ -36,6 +33,12 @@ pub enum AccountComponentInterface {
     /// Internal value holds the storage slot index where the public key for the RpoFalcon512
     /// authentication scheme is stored.
     RpoFalcon512(u8),
+    /// Exposes procedures from the
+    /// [`RpoFalcon512ProcedureAcl`][crate::account::auth::RpoFalcon512ProcedureAcl] module.
+    ///
+    /// Internal value holds the storage slot index where the public key for the
+    /// RpoFalcon512ProcedureAcl authentication scheme is stored.
+    RpoFalcon512ProcedureAcl(u8),
```

Comment:
```suggestion
    /// Internal value holds the storage slot index where the public key for the
    /// RpoFalcon512 authentication scheme is stored.
    RpoFalcon512ProcedureAcl(u8),
```

Nit: I would write just "RpoFalcon512" authentication scheme because the scheme is the same as the one used by the `RpoFalcon512` account component.

Reply:
Right, just force of habit: urge to rename everything with that name 🫠